files/image_config/ntp/ntp

```diff
@@ -50,11 +50,20 @@ case $1 in
 		fi
 		(
 			flock -w 180 9
-			vrfEnabled=$(/usr/local/bin/sonic-cfggen -d -v 'MGMT_VRF_CONFIG["vrf_global"]["mgmtVrfEnabled"]' 2> /dev/null)
+
+			# when mgmt vrf is configured, ntp starts in mgmt vrf by default unless user configures otherwise
+                        vrfEnabled=$(/usr/local/bin/sonic-cfggen -d -v 'MGMT_VRF_CONFIG["vrf_global"]["mgmtVrfEnabled"]' 2> /dev/null) 
+			vrfConfigured=$(/usr/local/bin/sonic-cfggen -d -v 'NTP["global"]["vrf"]' 2> /dev/null)
 			if [ "$vrfEnabled" = "true" ]
 			then
-				log_daemon_msg "Starting NTP server in mgmt-vrf" "ntpd"
-				cgexec -g l3mdev:mgmt start-stop-daemon --start --quiet --oknodo --pidfile $PIDFILE --startas $DAEMON -- -p $PIDFILE $NTPD_OPTS
+				if [ "$vrfConfigured" = "default" ]
+	                        then
+                                    log_daemon_msg "Starting NTP server in default-vrf for default set as NTP vrf" "ntpd"
+                                    start-stop-daemon --start --quiet --oknodo --pidfile $PIDFILE --startas $DAEMON -- -p $PIDFILE $NTPD_OPTS
+				else
+				    log_daemon_msg "Starting NTP server in mgmt-vrf" "ntpd"
+				    cgexec -g l3mdev:mgmt start-stop-daemon --start --quiet --oknodo --pidfile $PIDFILE --startas $DAEMON -- -p $PIDFILE $NTPD_OPTS
+				fi
```

Comment:
the indentation is strange for this file, please fix.

Reply:
The file seems to use tab spacing while I introduced single spaces. I will use tab space itself.

Reply:
Done.